presto-docs/src/main/sphinx/release/release-0.281.rst randomize_outer_join_null_key_strategy randomize_outer_join_null_key_strategy

```diff
@@ -0,0 +1,59 @@
+=============
+Release 0.281
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix timeout for optimization rules by adding interrupt check between each optimizer rule, and throw meaningful exception.
```

Comment:
This sounds confusing from an operator's perspective.  It's overly technical.
```suggestion
* Add timeouts for long running optimization rules.
```

Reply:
You probably missed out this one. Can you please double check @rohanpednekar